etl/steps/data/garden/state_capacity/2023-10-19/state_capacity_dataset.meta.yml

```diff
@@ -0,0 +1,447 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  state_capacity_dataset:
+    variables:
+      capacity:
+        title: State capacity estimate
+        unit: ""
+        short_unit: ""
+        description_short: "Measures state capacity by combining 21 different indicators related to three key dimensions: extractive capacity, coercive capacity, and administrative capacity. Higher values indicate greater state capacity."
+        description_from_producer: Description of state capacity from producer.
+        processing_level: minor
```

Comment:
It seems that the processing level is always minor. In that case, you could add at the beginning of the file:
```
definitions:
  common:
    processing_level: minor
```
And it will be applied to all indicators.

Reply:
Thanks, I filled that with copilot. I will wait if we decide to go for some regional aggregations, though.